### ISSUE TITLE
[Feature request] - "Quick Actions"

### ISSUE THREAD

**Issue by YouveGotMeowxy** [tannerhelland/PhotoDemon#424]
### Is this feature request related to a problem? 

Not a "problem" as such, but more of a super convenience.

### Describe the best way to implement this.

1. I would like to use PhotoDemon as a "quick, simple" editor as well as what it currently already is: an advanced editor similar to Photoshop.

Meaning, I'd like to for example, be able to screenshot something, (or download an image), and then add "quick actions" like a Highlighter tool, underline tool, quick pixilate/blur tool, quick text tool, quick arrows/outline (like a box, or circle) tool, etc.

As actual tool buttons in the toolbar to the left with the other tools. It seems that these features already exist when digging through all of the menus, but that's a bit cumbersome for "quick" stuff, and they're not easily/quickly repeatable w/out re-digging through the menu's again, which also means a large learning curve to find all of these, and always remember where they are and how to do them.

Basically similar to a tool like SnagIt.

2. As mentioned above, I also do a lot of screenshotting and pasting into editors; it would be very useful to have the "paste" feature added right as a button near the "open" button, so it's just a single, easy, intuitive click, rather than what I call "menu spelunking" :)

**Comment by tannerhelland** [comment]
This is a really interesting suggestion, @YouveGotMeowxy .  Because the left-hand toolbar is already getting crowded, I am intrigued by the idea of a single "quick edit" tool whose toolbar exposes a bunch of simple markup tools like you describe.  These could even be implemented as shortcuts to other existing tools (like a highlighter is basically just a semi-transparent paint brush) but as you say, it would exist more as a UI convenience for quick edits.

The 9.0 release is currently locked down against new features, but I will start studying this for inclusion post-9.0.  Thank you for the idea!

**Comment by hansened69-glitch** [comment]
is there a  quick;  create  'text box;  where i can select fonts, size,  colors right there once i drop down a text box to overlay on my jpg picture. 
it seems i have text to create then drop a box on top to edit all its features.  all the tools are there; just not as ez like adobe pdf comment editing, . See Faststone image viewer to quick overelay edits. 
again a great powerful program.  I just need to learn it.
